src/pages/SignUp/SignUp.js

```diff
@@ -0,0 +1,72 @@
+import { Link } from "react-router-dom";
+import "./SignUp.css";
+export function SignUp() {
+  return (
+    <>
+      <section className="signup-section section-center flex justify-center items-center">
+        <div className="signup-main-container flex flex-col items-center border-radius-2">
+          <h1 className="section-heading font-bold border-radius-1 p-2 my-2">
+            Signup
+          </h1>
+          <form>
+            <label className="input-label my-2">
+              First Name*
+              <input
+                type="text"
+                placeholder="Enter your first name"
+                className="input border-radius-1"
+                required
+              />
+            </label>
```

Comment:
As most of the code is repeating, you can add say `FormInput` component separately.

Reply:
Yup. will do in refactoring part.